modules/ws-client/package.json

```diff
@@ -5,7 +5,8 @@
     "postbuild": "cp -r static/* dist/"
   },
   "devDependencies": {
-    "parcel": "^2.3.2"
+    "parcel": "^2.3.2",
+    "process": "^0.11.10"
   },
   "dependencies": {
     "snabbdom": "^3.0.1"
```

Comment:
@gvolpe You should be able to remove this now. 😄

Reply:
The `snabbdom` dep, right? Can't remember how to manage `package.json` deps, do I just remove it manually?

Reply:
Yes, so in this case you could just remove the whole `dependencies` section manually, do a `yarn install`  then run it again and it should all still work just fine.

Reply:
That did it, thank you!